### PR TITLE
make example seed names more human friendly, and change imports

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: zingolib/src/wallet/disk/testing/tests.rs [hunk 1]
```diff
@@ -5,31 +5,20 @@ use zcash_client_backend::ShieldedProtocol;
 use zcash_keys::keys::Era;
 
 use crate::lightclient::LightClient;
+use crate::wallet::disk::testing::examples::{MobileShuffleVersion, VillageTargetVersion};
```
Comment (fluidvanadium) [inline zingolib/src/wallet/disk/testing/tests.rs]: should these be on line 18?

Comment (fluidvanadium) [reply zingolib/src/wallet/disk/testing/tests.rs]: this is evidence, to me, that this organization is worse.

Comment (zancas) [reply zingolib/src/wallet/disk/testing/tests.rs:8]: I considered this.   I think a better pattern is to collect the seeds into a `mod` named `seeds`.

I would be in favor of that.  I am not strongly committed to a particular import pattern **except** that I want them to be consistent, and I want code-actions to modify them without my intervention.

**Review by fluidvanadium — DISMISSED** [review]
zingolib/src/wallet/disk/testing/tests:

```
use crate::lightclient::LightClient;
use crate::wallet::disk::testing::examples::{MobileShuffleVersion, VillageTargetVersion};
use crate::wallet::keys::unified::UnifiedKeyStore;

use super::super::LightWallet;
use super::assert_wallet_capability_matches_seed;

use super::examples::NetworkSeedVersion;

use super::examples::{MainnetSeedVersion, RegtestSeedVersion, TestnetSeedVersion};

use super::examples::{
    AbandonAbandonVersion, AbsurdAmountVersion, ChimneyBetterVersion, HospitalMuseumVersion,
    HotelHumorVersion,
};
```

this is all over the place. while either paradigm is acceptable to me, it is essential that related items are grouped

File: zingolib/src/wallet/disk/testing/tests.rs (hunk 1 above)
Comment (zancas) [reply zingolib/src/wallet/disk/testing/tests.rs]: To me this is evidence that I, a human, am fighting against a tool-or-policy.   Again, I don't care much what the policy for imports is, but it should be consistent, and integrated with code-actions.

If humans are manually formatting import statements then they're wasting their time.

If a developer wants to change the policy for the project, that's fine they need to define the policy in the appropriate configuration, apply it uniformly, and open a PR with their new vision as the focus of the PR.

In that case please assign @Oscar-Pepper to review that PR.

I am sympathetic to a format that eases resolution of merge conflicts, but that format should not be applied in an ad-hoc way, and it should not be done without proper communication with stake-holders.

**Comment by fluidvanadium** [comment]
okay, if we are going to require that style, can we do it all the way instead of leaving it halfway?

im gonna do it now

**Comment by fluidvanadium** [comment]
merged conflict in #1486

fixed use statements to _**consistently**_ follow preferred style

**Comment by zancas** [comment]
Oh..  sorry, I kicked off a sanity test and forgot to check it pre-push..   then noticed it was done and pushed.   How's the current state?

File: zingolib/src/wallet/disk/testing/tests.rs [hunk 2]
```diff
@@ -5,32 +5,16 @@ use zcash_client_backend::ShieldedProtocol;
 use zcash_keys::keys::Era;
 
 use crate::lightclient::LightClient;
+use crate::wallet::disk::testing::examples::{
```
Comment (fluidvanadium) [inline zingolib/src/wallet/disk/testing/tests.rs]: please use import grouping consistently

**Review by fluidvanadium — CHANGES_REQUESTED** [review]
if a file has import grouping, all imports should be grouped

File: zingolib/src/wallet/disk/testing/tests.rs (hunk 2 above)
Comment (zancas) [reply zingolib/src/wallet/disk/testing/tests.rs]: How's this?

**Review by fluidvanadium — APPROVED** [review]
okay

**Pull request merged**